src/wallet/rpc/transactions.cpp

```diff
@@ -417,6 +421,10 @@ static std::vector<RPCResult> TransactionDescriptionString()
            }},
            {RPCResult::Type::STR_HEX, "replaced_by_txid", /*optional=*/true, "Only if 'category' is 'send'. The txid if this tx was replaced."},
            {RPCResult::Type::STR_HEX, "replaces_txid", /*optional=*/true, "Only if 'category' is 'send'. The txid if this tx replaces another."},
+           {RPCResult::Type::ARR, "mempoolconflicts", "Conflicting transaction currently in the mempool.",
```

Comment:
c71736d372c65b0978671501029e202574c6bbef

Also good to document - is it just directly conflicting, or descendants as well?

Reply:
Done